airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,30 +1,24 @@
 import React from "react";
-import sanitizeHtml from "sanitize-html";
+import DOMPurify from "dompurify";
 
 type IProps = {
   text?: string;
 };
 
-const allowedAttributes = {
-  ...sanitizeHtml.defaults.allowedAttributes,
-  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
-};
-
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
-  const sanitizedHtmlText = sanitizeHtml(text, {
-    allowedAttributes,
-    transformTags: {
-      a: sanitizeHtml.simpleTransform("a", {
-        target: "_blank",
-        rel: "noopener noreferrer",
-      }),
-    },
+  DOMPurify.addHook("afterSanitizeAttributes", function (node) {
+    if ("rel" in node) {
+      node.setAttribute("target", "_blank");
+      node.setAttribute("rel", "noopener noreferrer");
+    }
   });
 
+  var sanitizedHtmlText = DOMPurify.sanitize(text, { ALLOWED_TAGS: ["a"], ALLOWED_ATTR: ["rel"] });
```

Comment:
We're currently only allowing the `a` tag in this code and the `rel` attribute. I believe the default values of `sanitize-html` were different, and we had more tags allowed. Also with this change none of the links is actually working anymore, which most likely come from us only allowing the `rel` tag here. We should allow the exact same list that `sanitize-html` allowed (as default).